test/unit/test_environment.py

```diff
@@ -49,7 +49,8 @@
     'sagemaker_program':                   'main.py',
     'sagemaker_submit_directory':          'imagenet',
     'sagemaker_enable_cloudwatch_metrics': True,
-    'sagemaker_container_log_level':       logging.WARNING
+    'sagemaker_container_log_level':       logging.WARNING,
+    '_tuning_objective_metric':            'loss:3.4'
```

Comment:
nit: if you add a comma at the end of the line here, it'll make a cleaner diff the next time you need to add something to the dict

Reply:
Got it. I am working in a pylint file. I will add the the comma requirement when I enable linting.